.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: true
+contact_links:
+    - name: Send an e-mail to the developer
+      url: mailto:jgracia9988@gmail.com
```

Comment:
please put a placeholder, if you want to contact a developer it would be better if you do it through the github profile

Reply:
Considering this I think I will remove this cause this is kinda irrelevant.

Reply:
is fine with me